types/react-joyride/index.d.ts

```diff
@@ -9,15 +9,11 @@ import React = require("react");
 export default class Joyride extends React.Component<Props, State> {
     constructor(props: Props);
 
-    addTooltip(data: Step): void;
-    start(autorun?: boolean): void;
```

Comment:
This would especially be useful with the comment on the site:

```ts
    /**
     * Please don't use the `start` and `stop` methods anymore. Instead use a combination of the props `run` and `autoStart`.
     */
    private start(autorun? boolean): void;
```